pyiron_contrib/RDM/storagejob.py

```diff
@@ -0,0 +1,324 @@
+import os
+import posixpath
+import shutil
+
+from pyiron_base import GenericJob, DataContainer
+from pyiron_base.generic.filedata import FileData
+from pyiron_contrib.generic.s3io import FileS3IO
+
+
+def _remove_s3_working_directory(file_s3_io_handle, group_to_remove):
+    # Upon moving s3 to base, this should be moved to pyiron_base/job/util.py
+    if file_s3_io_handle.is_dir(group_to_remove):
+        file_s3_io_handle.remove_group(path=group_to_remove)
+
+
+class StorageType(DataContainer):
+    def __init__(self, *, init=None, storage_type=None, table_name=None):
```

Comment:
In this case I explicitly want to break the 'normal' `DataContainer` behavior:
usually when calling `DataContainer(something)` the `init` of `DataContainer` is set to `something` and the `DataContainer` is updated with these values. Here, I want to restrict to `kwargs` only.

Reply:
Behaviour sounds good to me. It would be nice to see that intent documented somewhere though, as it's not immediately obvious from the code. Maybe at the class-level docstring, to highlight difference to the parent class?